templates/CRM/Event/Form/ManageEvent/EventInfo.tpl

```diff
@@ -124,7 +124,7 @@
       </tr>
     {/if}
   </table>
-  {include file="CRM/common/customDataBlock.tpl" entityID=$eventID}
+  {include file="CRM/common/customDataBlock.tpl" entityID=$eventID cid=false}
```

Comment:
cid only needed for entities with more than one possible contact ID (activities)